pkg/kubectl/cmd/get.go

```diff
@@ -220,13 +222,23 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 	}
 
 	// use the default printer for each object
+	printer = nil
+	var lastMapping *meta.RESTMapping
+	w := tabwriter.NewWriter(out, kubectl.TabwriterMinWidth, kubectl.TabwriterWidth, kubectl.TabwriterPadding, kubectl.TabwriterPadChar, kubectl.TabwriterFlags)
```

Comment:
I'm not sure if there's a better way. We need to do `Flush()` only after visiting all objects. The tabwriter in HumanReadablePrinter.PrintObj() is different; it's for callers other than get.go, so that their output also get aligned.